pkg/metrics/metrics.go

```diff
@@ -253,8 +259,49 @@ var (
 		Subsystem: Datapath,
 		Name:      "errors_total",
 		Help:      "Number of errors that occurred in the datapath or datapath management",
-	},
-		[]string{LabelDatapathArea, LabelDatapathName, LabelDatapathFamily})
+	}, []string{LabelDatapathArea, LabelDatapathName, LabelDatapathFamily})
+
+	// ConntrackGCDeleted is the number of deleted entries in the
+	// conntrack table that has expired.
+	ConntrackGCDeleted = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Namespace: Namespace,
+		Subsystem: Datapath,
+		Name:      "conntrack_gc_deleted",
+		Help:      "Number of deleted entries in the conntrack table by the garbage collector process",
+	}, []string{LabelDatapathFamily, LabelScope})
+
+	// ConntrackGCRuns is the number of times that the conntrack GC
+	// process was run.
+	ConntrackGCRuns = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Namespace: Namespace,
+		Subsystem: Datapath,
+		Name:      "conntrack_gc_runs_total",
+		Help:      "Number of times that the conntract garbage collector process was run",
+	}, []string{LabelDatapathFamily, LabelCompleted})
+
+	// ConntrackGCKeyFallbacks number of times that the conntrack key fallback was invalid.
+	ConntrackGCKeyFallbacks = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Namespace: Namespace,
+		Subsystem: Datapath,
+		Name:      "conntrack_gc_key_fallbacks",
+		Help:      "Number of times that the Key fallback was invalid.",
+	}, []string{LabelDatapathFamily})
+
+	// ConntrackGCSize the number of entries in the conntrack table
+	ConntrackGCSize = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: Namespace,
+		Subsystem: Datapath,
+		Name:      "conntrack_gc_size",
+		Help:      "The size of the conntrack table when the garbage collector ran",
```

Comment:
Could we be even more specific here? "The number of entries in the conntrack table at the *start* of a GC run"

Reply:
Oh, I noticed that gcStats has an aliveEntries field. Can we use that with label here? or is it guaranteed that aliveEntries is the final number left in the table?

Also, I don't see where `ConntrackGCSize` is set